comprehensive/lms/templates/user_dropdown.html

```diff
@@ -55,7 +53,9 @@
             <div role="group" aria-label="User menu" class="user-menu">
                 <button class="dropdown" aria-expanded="false"><span class="sr">${_("More options dropdown")}</span><span class="fa fa-sort-desc" aria-hidden="true"></span></button>
                 <ul class="dropdown-menu" aria-label="More Options" role="menu">
-                    ${navigation_dropdown_menu_links()}
+                    % if request.get_full_path() not in ['/account/link', '/account/link/confirm']:                        
```

Comment:
I haven't tested get_full_path() in a while. Does get_full_path() include query parameters (after question mark). I ask because that might be a potential strategy for getting around this conditional if it does.
FWIW: I know fragments (after the hash symbol) aren't sent to the server so we don't have to worry about that

Reply:
Good catch